src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -51,6 +59,25 @@ class MemberPermission(OrganizationPermission):
     }
 
 
+class OrganizationGroupSearchViewGetSerializer(serializers.Serializer[None]):
+    visibility = serializers.ListField(
```

Comment:
you should be able to use `MultipleChoiceField` like in `https://github.com/getsentry/sentry/blob/352c4f86082ce805fd4904a0c8fb8fe262447e1d/src/sentry/integrations/api/serializers/rest_framework/doc_integration.py#L45

which simplifies the logic a bit